modules/pubmaticBidAdapter.js

```diff
@@ -217,6 +219,108 @@ function _checkParamDataType(key, value, datatype) {
   }
 }
 
+function _createNativeRequest(params) {
+  var nativeRequestObject = {
+    assets: []
+  };
+  for (var key in params) {
+    var assetObj = {};
+    if (!(nativeRequestObject.assets && nativeRequestObject.assets.length > 0 && nativeRequestObject.assets.hasOwnProperty('key'))) {
+      switch (key) {
+        case 'title':
+          assetObj = {
+            id: 1,
+            req: params[key].required,
+            title: {
+              len: params[key].length,
+              ext: params[key].ext
+            }
+          };
+          break;
+        case 'image':
+          assetObj = {
+            id: 2,
+            req: params[key].required,
+            img: {
+              type: params[key].type,
+              w: params[key].w || params[key].width || params[key].sizes[0],
```

Comment:
list-down mandatory and non-mandatory params

Reply:
need validation on mandatory params

Reply:
created a minimum mandatory object required by prebid to send bid to image type native ad.

Validation to be done at the time of response ?